src/Zynga/Framework/ReflectionCache/V1/ReflectionClasses.hh

```diff
@@ -6,17 +6,16 @@ use \Exception;
 use \ReflectionClass;
 
 class ReflectionClasses {
```

Comment:
If we made this a non static class and instead used a factory method to return a particular implementation, I think we can get away from having to inject mocking code into these classes. But that's not something for now.